01/components/Table/index.js

```diff
@@ -0,0 +1,7 @@
+import Table from './Table';
+import { TableHeader } from './TableHeader';
+import { TableBody } from './TableBody';
+import { TableRow } from './TableRow';
+import { TableFooter } from './TableFooter';
+
+export { Table, TableHeader, TableBody, TableRow, TableFooter };
```

Comment:
To zadanie jest wykonane poprawnie, jednak można by było się pokusić o bardziejuniwersalne.
Idealnie gdybyśmy mogli przekazać informacje o kolumnach i już na tej podstawie będzie wygenerowana zwartość tej tabeli. Tutaj potrzebujemy nazwę kolumny oraz nazwę właściwości z jakiej trzeba pobrać wartość np. 
```
<Table columns={[{label: 'Cena', name: 'price'}, {label: 'Nazwa', name: 'name'}]} data={data}/>
```

Traktuj to ao zadanie dodatkowe - jak znajdziesz czas to ok, jak nie to też nic się nie stanie ;)